src/components/renderer/form-analytics-chart.vue

```diff
@@ -41,10 +43,14 @@ export default {
     if (this.listChartOption && this.listChartOption.name) {
       this.graphic = this.listChartOption;
     }
+    this.verifyChart();
   },
   methods: {
     openExternalLink() {
       window.open("/package-analytics-reporting", "_blank");
+    },
+    verifyChart() {
+      this.showChart = this.graphic.name === "PM Analytics Chart" || this.graphic.name === "PM Analytics Dashboard";
```

Comment:
Please this default needs to show when the Analytics does not have any chart selected like: 
![image](https://github.com/ProcessMaker/screen-builder/assets/42388723/60579a40-346d-41ec-a197-db8aa4b42766)